### ISSUE TITLE
Install.sh

### ISSUE THREAD

**Issue by hmatic** [cyberark/summon#85]
After #84 I can no longer install summon using install.sh, it also fails installing it on docker container during deployment.
It stops at do_download and returns status code 8.
Do you guys know what's the issue? Is there any easy way to manage installed summon versions in Dockerfile with ubuntu image?

**Comment by sgnn7** [comment]
Hey @hmatic, thank you for reporting the issue!

 Can you provide us with a bit more info on your environment:
- Operating system
- Architecture
- Is curl installed?
- Is wget installed?

It would be great if you could provide the full log of the docker run and the installation attempt too so that we can make sure that we're looking into the right place.

I should be able to take a look at this some point today if you get me these things.

**Comment by sgnn7** [comment]
@hmatic The [release of v0.6.9](https://github.com/cyberark/summon/releases/tag/v0.6.9) seems to have fixed issues with `./install.sh` here locally. Please try again and reopen the issue if you still have problems. Thanks!

**Comment by sgnn7** [comment]
You cannot do it directly but you can download [install.sh](https://github.com/cyberark/summon/blob/master/install.sh) script manually and change [this](https://github.com/cyberark/summon/blob/6e8bba50d717ec85536a0e26cbe2dc97c9a07e06/install.sh#L54) line to say `LATEST_VERSION="0.6.8"`. Feel free to open a new issue for this as well as it seem like a good feature to have.
